badges/badges.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="badges.css">
+    <link rel="stylesheet" href="../components.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Document</title>
+</head>
+<body>
+  
+    
+    <!--navbar-->
+    <div class="navbar">
+        <div class="nav-items">
+            <a class="nav-heading" href="../index.html"><img src="../images/logo.png"></a>
+        </div>
+    </div>
+
+    <!--Main section-->
+    <section>
+        <div class="container flex-flow-row">
+            <div class="sidebar flex-flow-column">
+                <h2 class="sidebar-heading"><a>Getting started</a></h2>
+                <a href="" class="component-items">Installation</a>
+                <a href="../avatar/avatar.html" class="component-items">Avatars</a>
+                <a href="../alerts/alerts.html" class="component-items">Alerts</a>
+                <a href="../buttons/buttons.html" class="component-items">Buttons</a>
+                <a href="../cards/cards.html" class="component-items">Card</a>
+                <a href="badges.html" class="component-items">Badges</a>
+                <a href="../image/image.html" class="component-items">Image</a>
+                <a href="../input/input.html" class="component-items">Input</a>
+            </div>
+
+            <div class="avatar-container component-container">
+                <div class="component-details">
+                    <h1 class="component-heading">Badges</h1>
+                    <p class="component-info">Documentation for badges, a labeling component.Badges scale to match the size of the immediate parent element by using relative font sizing.</p>
+                </div>
+                <div class="avatar-examples avatar-images flex-flow-row">
+                    <div class="badge-container">
+                        <img class="avatar avatar-lg" src="../images/avatar.jpg">
```

Comment:
Just a suggestion add an **alt attribute** on images basically it helps to describe the images

Reply:
Thanks, will make the changes